examples/tutorials/translations/german/Part 01 - The Basic Tools of Private Deep Learning.ipynb

```diff
@@ -0,0 +1,966 @@
+{
```

Comment:
<p>Herzliche Glückwünsche should be "Herzlichen Glückwunsch", I just noticed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/3178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>